app/src/main/java/org/hackillinois/android/view/IndoorMapsFragment.java

```diff
@@ -24,6 +27,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         tabs.addOnTabSelectedListener(new TabListener());
 
         map = view.findViewById(R.id.map);
+        map.setMinimumTileDpi(160);

```

Comment:
Where did 160 come from?

Reply:
It's arbitrary. A comment in the source of the library said that 160-240 should be acceptable and would make images load faster. By default it is something in the 300s, and I didn't see much quality degradation by switching to this. Maybe I could create a variable called MINIMUM_TILE_DPI or something? What do you think?

Reply:
sounds good to me, +1 for constant